content/blog/2021-10-05-extending-models-with-more-data.md

```diff
@@ -0,0 +1,256 @@
+```
+
+The `--no-timestamp` hides the timestamps from table. The `--includes-metrics`
+option lets us choose which metrics we want to show in the table. The
+`--includes-params` option does the same for hyperparameters. This gives us a
+table that's easier to read quickly.
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━┳━━━━━━━━━━┓
+┃ neutral:**Experiment**              ┃ metric:**step** ┃     metric:**acc** ┃ metric:**val_acc** ┃    metric:**loss** ┃ metric:**val_loss** ┃ param:**lr**    ┃ param:**momentum** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━╇━━━━━━━━━━┩
+│ **workspace**               │    **3** │ **0.86885** │    **0.46** │ **0.31573** │   **3.7067** │ **0.001** │ **0.09**     │
+│ **data-change**             │    **-** │       **-** │       **-** │       **-** │        **-** │ **0.001** │ **0.09**     │
+│ │ ╓ 3b3a2a2 [exp-23593] │    3 │ 0.86885 │    0.46 │ 0.31573 │   3.7067 │ 0.001 │ 0.09     │
+│ │ ╟ 93d015d             │    2 │ 0.83197 │ 0.41333 │ 0.36851 │   3.4259 │ 0.001 │ 0.09     │
+│ │ ╟ d474c42             │    1 │ 0.79918 │ 0.43333 │ 0.46612 │    3.286 │ 0.001 │ 0.09     │
+│ ├─╨ 1582b4b             │    0 │ 0.52869 │    0.39 │ 0.94102 │   2.5967 │ 0.001 │ 0.09     │
+└─────────────────────────┴──────┴─────────┴─────────┴─────────┴──────────┴───────┴──────────┘
+```
+
+You'll notice that the validation accuracy is really low. That's because the
+training metrics are based on bees and ants while the validation metrics are
+based on bees, ants, cats, and dogs. If we looked at the validation metrics by
+class, they'd likely be better for bees and ants than cats and dogs.
+
+That means we should probably add more data to the training dataset.
+
+## Adding the cats data to the training dataset
+
+Let's add the `train` data for `cats` to the corresponding directory in
+`data/hymenoptera_data` and go through another experiment run with a different
+learning rate. With this new data, we can run another experiment. One important
+thing to note here is that we're using checkpoints in our experiments. That's
+how we get the metrics for each training epoch.
+
+If we want to run a fresh experiment that doesn't resume training from the last
+epoch, we need to reset our experiment. That's what we're going to do with this
+command.
+
+```dvc
+dvc exp run --reset
+```
+
+This will reset all of the existing checkpoints and excute the training script.
+Once it's finished, let's take a look at the metrics table with this command.
+It's the same as the one we ran last time.
+
+```dvc
+dvc exp show --no-timestamp --include-metrics step,acc,val_acc,loss,val_loss --include-params lr,momentum
+```
+
+Now you'll have a table that shows both experiments and you can see how much
+better the new one did with the `cats` data added.
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━┳━━━━━━━━━━┓
```

Comment:
btw, this is a good example where we want to see in the table changes to data I think? cc @dberenbaum